bash/01_dataloading.sh

```diff
@@ -5,7 +5,7 @@ source bash/config.sh
 MODE="${1:-edm}"
 
 max_bg_procs 5
-import_public dob_now_applications &
+import_public dob_now_applications 20220921 &
```

Comment:
Should this have been committed?

Reply:
It was set to this date because this is the only data from dob that includes these new columns, I can take it out before merging into dev but if you are testing it you'll need to set this date to see if the data populates these new column additions

Reply:
sure that makes sense. I think the way to handle this situation is to put direction on how to run in the PR but not commit this change. Committing it suggests that you want it merged in, which we definitely don't want

Reply:
Thought I kind of prompted this discussion in the PR write up but will be more explicit next time and not include the changes we don't want to commit